src/home_robot_hw/home_robot_hw/utils/grasping.py

```diff
@@ -291,7 +224,6 @@ def try_executing_grasp(
         """Execute a predefined grasp trajectory to the given pose. Grasp should be an se(3) pose, expressed as a 4x4 numpy matrix."""
         assert grasp.shape == (4, 4)
         self._send_predicted_grasp_to_tf(grasp)
-        wait_for_input = True
```

Comment:
`wait_for_input` is used in this function. It might give an error since the variable is not present in the function anymore.

Reply:
it's passed in as a function parameter

Reply:
Sorry my mistake